_posts/2021-06-02-#21527.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: pr
+date: 2021-06-02
+title: "Net_processing: lock clean up"
+pr: 21527
+authors: [ajtowns]
+components: ["p2p", "refactoring"]
+host: jnewbery
+status: upcoming
+commit:
```

Comment:
needs a commit?

Reply:
The commit hash isn't used until the status changes to "past" (so it is often done when doing that and adding the meeting log). Also, the PR author might re-push during the days before the meeting and (sometimes) it makes sense to use the latest one.